qfieldcloud_sdk/sdk.py

```diff
@@ -617,6 +617,33 @@ def job_status(self, job_id: str) -> Dict[str, Any]:
 
         return resp.json()
 
+    def push_delta(self, project_id: str, delta_filename: str) -> Dict[str, Any]:
+        """Push a delta file to a project.
+
+        Args:
+            project_id: Project ID.
+            delta_filename: Path to the delta JSON file.
+
+        Returns:
+            A dictionary containing the response from the server.
+        """
+        try:
+            with open(delta_filename, "r") as delta_file:
+                files = {"file": delta_file}
+                resp = self._request(
+                    "POST",
+                    f"deltas/{project_id}/",
+                    files=files,
+                )
+            print(f"Response status code: {resp.status_code}")
```

Comment:
```suggestion
            print(f"Response status code: {resp.status_code}")
```
Avoid all the prints. If you need to log something, please use the logger.

This print is writing to STDOUT which is problematic as STDOUT is for the actual result (e.g. response JSON).

Reply:
I think is better to completely remove them